app.js

```diff
@@ -1,7 +1,9 @@
 const express = require("express");
 const app = express();
+const apiRouter = require("./routes/apiRoutes");
 const topicsRouter = require("./routes/topicsRoutes");
 
+app.use("/api", apiRouter);
```

Comment:
Looking at the line below this. You can and probably should nest routers ( not relevant to this ticket but important.)